src/InstagramClone.Api/Startup.cs

```diff
@@ -0,0 +1,36 @@
+﻿using InstagramClone.Api.Database;
+using Microsoft.EntityFrameworkCore;
+
+namespace InstagramClone.Api
+{
+    public static class Startup
+    {
+        public static WebApplication InitialIzeApp(string[] args)
+        {
+            var builder = WebApplication.CreateBuilder(args);
+            ConfigureService(builder);
+            var app = builder.Build();
+            Configure(app);
+            return app;
+        }
+
+        private static void ConfigureService(WebApplicationBuilder builder)
+        {
+            
+            builder.Services.AddDbContext<InstagramCloneDbContext>(options =>
+            {
+                options.UseSqlServer(builder.Configuration.GetConnectionString("DefaultConnectionString"));
+
+               
+            });
```

Comment:
The error is because you're configuring DbContext twice. 

You are configuring it here...